### PR TITLE
feat: add australia region support

### PR DESCRIPTION
**Pull request by bitnimble**
This PR adds support for Kia and Hyundai in Australia. It is largely a copy of the China + Europe authentication mechanisms, with some changes to the vehicle parser (seems like we get a little less information :'( )

I will probably look at consolidating the vehicle parsers at some stage now that we've hit ~3 identical implementations, but I need a bit more time actually using it to make sure that everything is correct.

### CODE REVIEW

File: debug.ts [hunk 1]
```diff
@@ -45,20 +45,20 @@ const askForRegionInput = () => {
         type: 'list',
         name: 'region',
         message: 'What Region are you in?',
-        choices: ['CN','US', 'EU', 'CA'],
+        choices: ['CN', 'US', 'EU', 'CA', 'AU'],
```
Comment (bitnimble) [inline debug.ts:48]: This is the only change in this file, everything else is from Prettier

File: src/constants/stamps.ts [hunk 2]
```diff
@@ -15,7 +20,8 @@ const cachedStamps = new Map<string, StampCollection>();
 
 const getAndCacheStampsFromFile = async (
```
Comment (bitnimble) [inline src/constants/stamps.ts:21]: As Europe and Australia will share a remote stamp generator, I've renamed this file and just added `stampHost` as a param for europe/australia brand configs to pass in instead.

For Europe it'll be neoPix's repo, for Australia it'll be mine.

File: src/util.ts [hunk 3]
```diff
@@ -24,8 +24,14 @@ const REGION_STEP_RANGES = {
   CN: {
     start: 14,
     end: 30,
-    step:0.5
-  }
+    step: 0.5,
+  },
+  // TODO: verify the Australian temp code ranges
```
Comment (bitnimble) [inline src/util.ts:29]: I'm like 90% sure this is correct, but will need to validate more in the future with some other vehicles just in case.

**Comment by Hacksore** [comment]
Can you give this a test @bitnimble and we can :shipit: if all is working well.

I did just publish `hacksore/hks:native-au`.
https://hub.docker.com/layers/hacksore/hks/native-au/images/sha256-b564d6d6c6d18a8a4e2e76d50d69051cb65e9c8b0b45a7f2dfab8baf6011d56e?context=repo

**Comment by bitnimble** [comment]
Tested and working! It did generate a new CFB, but both the old and new ones work, so I'm not really sure what's up there...

**Pull request merged**